store/shared/migrate/mysql/ddl_gen.go

```diff
@@ -186,6 +186,7 @@ func selectCompleted(db *sql.DB) (map[string]struct{}, error) {
 
```

Comment:
this file is automatically generated and should not be manually updated. Otherwise the next time we generate this file your changes will be lost. You need to send a pull request to update the generator, and then you need to regenerate this file using `go generate`

Reply:
sorry I don't know the file where should I modify.

Reply:
I fond.
https://github.com/drone/drone-migrate/blob/master/migrate/db/mysql/ddl_gen.go